packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -282,6 +298,16 @@ const MainScreen = (props: MainScreenProps): JSX.Element => {
         </>
       );
       break;
+    case unsupportedBrowserPageTrampoline():
```

Comment:
Personally, I would prefer an if statement outside this switch statement without the trampoline. But this works.